dace/sdfg/state.py

```diff
@@ -1226,6 +1214,45 @@ def add_reduce(
         self.add_node(result)
         return result
 
+    def add_pipeline(
+            self,
+            name,
+            ndrange,
+            init_size=0,
+            init_overlap=False,
+            drain_size=0,
+            drain_overlap=False,
+            schedule=dtypes.ScheduleType.FPGA_Device,
+            debuginfo=None,
+            **kwargs
+    ) -> Tuple[nd.PipelineEntry, nd.PipelineExit]:
+        """ Adds a pipeline entry and pipeline exit.
```

Comment:
Add a better definition of what a pipeline is? Doesn't have to be long

Reply:
dun